editor.planx.uk/src/lib/graphql.ts

```diff
@@ -131,11 +127,48 @@ const retryLink = new RetryLink({
   },
 });
 
+/**
+ * Set auth header in Apollo client
+ * Must be done post-authentication once we have a value for JWT
+ */
+export const authMiddleware = setContext(async () => {
+  const jwt = await getJWT();
+
+  return {
+    headers: {
+      authorization: jwt ? `Bearer ${jwt}` : undefined,
+    },
+  };
+});
+
+/**
+ * Get the JWT from the store, and wait if not available
+ */
+const getJWT = async () => {
```

Comment:
I did spend a bit of time trying to make this a function in the `UserStore` but couldn't get the `subscribe()` on line 159 to work within this context.